### PR TITLE
Clean up parameters intro.

### PR DESCRIPTION
**Pull request by Rowlando13**
Made the Parameters' first few sections cleaner and more concise.

### CODE REVIEW

File: docs/parameters.rst [hunk 1]
```diff
@@ -3,39 +3,60 @@ Parameters
...
-On the other hand arguments, unlike options, can accept an arbitrary number
-of arguments.  Options can strictly ever only accept a fixed number of
-arguments (defaults to 1), or they may be specified multiple times using
-:ref:`multiple-options`.
+Click supports only two types  of parameters for scripts (by design): options and arguments.
+
+Options
+----------------
+
+*   are optional.
```
Comment (tachyondecay) [inline docs/parameters.rst:11]: Capitalize "Are" for consistency in the list.

File: docs/parameters.rst [hunk 2]
```diff
@@ -3,39 +3,60 @@ Parameters
...
+*   Can take a fixed number of arguments. The default is 1. They may be specified multiple times using :ref:`multiple-options`.
+*   Are fully documented by the help page.
+*   Have automatic prompting for missing input.
+*   Can act as flags (boolean or otherwise).
+*   Can be pulled from environment variables.
+
+Arguments
+----------------
+
+*   are optional with in reason, but not entirely so.
```
Comment (tachyondecay) [inline docs/parameters.rst:22]: Capitalize "Are" for consistency in the list.

**Review by tachyondecay — COMMENTED** [review]
Looks good overall—some of the lines aren’t wrapped, not sure how much this matters.

**Comment by Rowlando13** [comment]
Fixed as @tachyondecay suggested. I generally don't split rst files at the 80 character limit since you can just use the text wrap setting. Documentation is long form writing so 80 characters does not make a lot of sense.

**Pull request merged**